packages/package-bundler/src/presets/default/plugins-default.ts

```diff
@@ -0,0 +1,40 @@
+import alias, { RollupAliasOptions } from "@rollup/plugin-alias"
+import { babel, getBabelOutputPlugin } from "@rollup/plugin-babel"
+import { InputPluginOption } from "rollup"
+import ignore from "rollup-plugin-ignore"
+import nodeExternals from "rollup-plugin-node-externals"
+
+type Config = {
+  alias: RollupAliasOptions | undefined
+}
+
+export const presetDefault = (config: Config): InputPluginOption[] => [
+  nodeExternals({
+    devDeps: true,
+  }),
+  // Has to be the same as packages/components/tsconfig.json -> compilerOptions -> paths
+  alias(config.alias),
+  // This call to alias plugin will be additional to the above alias plugin call
+  alias({
+    entries: [
+      // i18n-react-intl package attempts to import locales from this path.
+      // When rollup attempts to import from the 'find' path, it will be
+      // redirected to import from the replacement path (same as storybook webpack config).
+      {
+        find: "__@cultureamp/i18n-react-intl/locales",
+        replacement: "locales",
+      },
+    ],
+  }),
+  // These libraries aren't used in KAIO, and require polyfills to be set up
+  // in consuming repos. Ignoring them here removes the need for extra setup in
+  // consuming repos.
+  ignore(["stream", "http", "https", "zlib"]),
```

Comment:
should we include also other Nodejs modules similarly as we do for [example here](https://github.com/cultureamp/unified-home/blob/master/packages/next-storybook/src/addons/sensibleDefaults/preset.ts#L53-L66)?